packages/client/tests/ecosystem/_utils/standard.cmd.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/sh
+
+(
```

Comment:
This script copies files (tarballs, tests, utils) into the container from the shared volume and prepares the test, puts it into `/test` while also logging its output into a file.